code/style.css

```diff
@@ -1,4 +1,102 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box;
 }
+
+body {
+  font-family: "Open Sans Condensed", sans-serif;
+  background: rgb(224, 190, 224);
+  margin; 0;
+}
+
+ /*Navbar*/
+nav{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px;
+}
+
+.smiley {
+  display: flex;
+}
+
+.logo {
+  width:50px;
+  height:50px;
+}
+
+h1{
+  margin: 5px 10px;
+}
+
+a{
+  margin: 10px;
+  color: black;
+  text-decoration: none;
+}
+a:hover {
+font-weight: bold;
+}
+
+/*Big News*/
+.big-news {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+   background-image: url("peace.jpg");
+   background-repeat: no-repeat;
+   background-position: center center;
+   background-size: cover;
+   border: 3px solid black;
+   padding: 250px 40px;
+   margin: 0px 60px;
+}
+
+hr{
+  border: 1px solid black;
+  margin-top:20px;
+  margin-left: 12px;
+  margin-right:12px;
+}
+/*Small New*/
+.small-news {
+display: flex;
+flex-wrap: wrap;
+justify-content: center;
+}
+
+.smallstory{
+  border: 1px solid black;
+  max-width: 24%;
+  margin: 20px 5px;
+  background: rgb(218, 214, 214);
+  box-shadow: 2px 2px rgb(167, 165, 165) ;
+  text-align: center;
+}
+
+.smallstory:hover {
+  background-color: rgb(224, 190, 224);
+}
+
+.articletext{
+  padding: 15px;
+  text-align: center;
+}
+
+.articleimage{
+  max-width: 100%;
+  max-height: 150px;
+}
+
+@media only screen and (max-width: 768px) {
+  .smallstory {
+    max-width: 49.99999%;
```

Comment:
how come you went for `max-width: 49.99999%;`?

Reply:
An example on I think W3-schools used it. I thought it would be good since it wouldn't cover the whole screen then and I wouldn't have to use any extra margins or padding. Should I be seeing it some other way?

Reply:
Okay, just wanted to hear how you decided that. I would say better to go with 50% or 49%. but if this is whats working best, keep it.